.github/comments-aggregator/README.md

```diff
@@ -0,0 +1,57 @@
+# comments-aggregator
+
+> This GitHub Action helps you keep the PR page clean by merging comments/reports
+> by multiple workflows into a single comment.
+
+![screenshot](./screenshot.png)
+
+## Usage
+
+This action is meant is be used as the poster/commenter. Instead of having
+existing actions post the comment by themselves, set those comments as the
+action output, then feed that output to `comments-aggregator` to let this action
+manage those comments for you.
+
+```yml
+            - name: Compare Assets
+              uses: ./.github/compare-assets
+              id: compare-assets
+              with:
+                  repo-token: '${{ secrets.GITHUB_TOKEN }}'
+                  compare: assets-list/assets.json
+                  create-comment: false
+            - name: Append report
+              uses: ./.github/comments-aggregator
+              with:
+                  repo-token: '${{ secrets.GITHUB_TOKEN }}'
+                  section-id: compare-assets-with-trunk
+                  content: ${{steps.compare-assets.outputs.comment}}
+```
```

Comment:
Nit: I would move this code block one indentation level up, to match the code block below.